javascript/extractor/src/com/semmle/js/extractor/AutoBuild.java

```diff
@@ -1239,11 +1240,29 @@ private void logEndProcess(long timedLogMessageStart, String message) {
   protected void extractXml() throws IOException {
     if (xmlExtensions.isEmpty()) return;
     List<String> cmd = new ArrayList<>();
-    cmd.add("odasa");
-    cmd.add("index");
-    cmd.add("--xml");
-    cmd.add("--extensions");
-    cmd.addAll(xmlExtensions);
+    if (EnvironmentVariables.getCodeQLDist() == null) {
+      // Use the legacy odasa XML extractor
+      cmd.add("odasa");
+      cmd.add("index");
+      cmd.add("--xml");
+      cmd.add("--extensions");
+      cmd.addAll(xmlExtensions);
+    } else {
+      String command = Env.getOS() == OS.WINDOWS ? "codeql.cmd" : "codeql";
+      cmd.add(EnvironmentVariables.getCodeQLDist() + "/" + command);
```

Comment:
Shouldnt `"/"` be something like `File.separator`?
 (or maybe just use something like `Path.resolve` instead, as done elsewhere in this file)

Reply:
I'd be in favour of using `Paths.get(EnvironmentVariables.getCodeQLDist(), command)` or indeed `Paths.resolve()`